### DISCUSSION TITLE
How do I create a domain with just a username?

### DISCUSSION THREAD

**Discussion by frazrabotka** [community/community#142949] [GitHub Education] [answered]
### Hi everyone,

Hi. 
I want to create a website on github that will look like https://frazrabotka.github.io To do this, as I understand it, you need the repository to match the username. I did that. But still the site only works at this address https://frazrabotka.github.io/frazrabotka/ How to set up the site correctly so that it looks like https://frazrabotka.github.io Thank you. (https://github.com/frazrabotka/frazrabotka)
Thanks.
![2](https://github.com/user-attachments/assets/fcb011a1-1367-4e7e-81c4-47415a3a4666)
![11](https://github.com/user-attachments/assets/d1e67266-7ee2-41a1-b74a-e015c1e4cf18)

**Comment by davevad93** [comment, marked as answer]
Hi @frazrabotka ,

To set up your `username.github.io` page you have to name the repository `frazrabotka.github.io`.

The repo that matches the GitHub `username` is used for the [profile README](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)

**Reply by frazrabotka** [reply]
Everything worked out. thanks
